Engine/SerializerPlugins/TestStepSerializer.cs

```diff
@@ -56,11 +56,20 @@ public void FixupStep(ITestStep step, bool recurse)
         {
             if (stepLookup.TryGetValue(step.Id, out ITestStep currentStep) && currentStep != step && !ignoredGuids.Contains(step.Id))
             {
+                ITestStepParent getTestPlanReferenceParent(ITestStep s) =>
+                    s.GetParents().FirstOrDefault(p => p.GetType().Name == "TestPlanReference");
```

Comment:
Maybe we should add an interface type for steps that load their own child steps? That way we won't have to do this string comparison.